src/Controls/tests/Core.UnitTests/AppThemeTests.cs

```diff
@@ -34,6 +34,7 @@ public void ThemeChangeUsingSetAppThemeColor()
 			{
 				Text = "Green on Light, Red on Dark"
 			};
+			app.LoadPage(new ContentPage {Content = label});
```

Comment:
apotheme propagation REQUIRES that the control is parented, which is fine, as otherwise it wouldn't be on screen